metaflow/plugins/kfp/kfp.py

```diff
@@ -178,6 +191,16 @@ def __init__(
         self.notify_on_success = notify_on_success
         self._client = None
 
+        self._compile_time_bootstrap_cmd = self._get_metaflow_bootstrap_cmd(
+            code_package_url=code_package_url,
+            environment=environment,
+        )
+        self._run_time_bootstrap_cmd = self._get_metaflow_bootstrap_cmd(
+            code_package_url=code_package_url,
+            environment=environment,
+            run_time=True,
+        )
```

Comment:
This is at the wrong level of abstraction.

Reply:
Changed. I have made it so that:
- `MetaflowBootstrapVars` is its own class (outside of `KfpComponent`). Moreso, `MetaflowBootstrapVars` contains variables that are consistent across the flow (e.g. `tags`). Variables `KfpComponent` are step-specific (e.g. `step_name`).